src/frontend/components/StudentLiveWrapper/index.tsx

```diff
@@ -72,7 +81,10 @@ export const LiveVideoWrapper: React.FC<LiveVideoWrapperProps> = ({
         isPanelOpen={isPanelVisible}
         isXmppReady={!!video.xmpp}
         liveTitleElement={
-          <StudentLiveInfoBar title={video.title} startDate={null} />
+          <StudentLiveInfoBar
+            title={video.title ?? intl.formatMessage(messages.defaultLiveTitle)}
```

Comment:
```suggestion
            title={video.title || intl.formatMessage(messages.defaultLiveTitle)}
```
IMO, if the title has for value an empty character chain, we should prefer display the `messages.defaultLiveTitle` instead of nothing. (But I don't know if this case can really happen, because when the live is created it has no title so it's `null` or `undefined`. I think it will depend on how we want to implement the title modification and on if an instructor can remove the title of its video or not)

Reply:
I disagree, if we want to consider an  empty string as a null title, the title should be trimmed in the bakend and set in the video accordingly